source/admin_manual/list_of_events.rst

```diff
@@ -1286,6 +1286,28 @@ A field's caching decision changed. The decisions are:
 | new_decision        | New cache decision: no, temp, yes                      |
 +---------------------+--------------------------------------------------------+
 
+.. _event_mail_cache_decision_rejected:
+
+mail_cache_decision_rejected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The decision to promote a field (from ``no`` to ``temp``) was rejected. This can
+happen when the count of headers in the cache exceeds the maximum configured with
+:dovecot_core:ref:`mail_cache_max_headers_count`.
+
++---------------------+--------------------------------------------------------+
+| Field               | Description                                            |
++=====================+========================================================+
+| field               | Cache field name (e.g. ``imap.body`` or ``hdr.from``)  |
++---------------------+--------------------------------------------------------+
+| last_used           | UNIX timestamp of when the field was accessed the last |
+|                     | time. This is updated only once per 24 hours.          |
```

Comment:
This doesn't seem useful. Since the decision is "no" at this stage, the last_used must have been set in the current session, so it's not much different from the current time.